### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ngodi**

### CODE REVIEW

File: spec/em_spec.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      it "returns false if no every element fills the condition" do
+        expect([0,0,0,0,2].my_all?{|n| n == 0}).to eql(false)
+      end
+    end
+end
+
+
+describe "#my_each_with_index" do
+    it "returns every array element with it's index" do
+    expect([9,2,4].my_each_with_index {|element, index| [element, index]}).to eql([[9, 0], [2,1], [4,2]])
```
Comment (inhaq) [inline spec/em_spec.rb]: as you're using the same array multiple times, kindly use `let` to avoid repetitions

**Review by inhaq — CHANGES_REQUESTED** [review]
Good work, team. Just minor improvement and you're good to go. Once fixed, kindly resubmit.

File: spec/em_spec.rb (hunk 1 above)
Comment (diegoarvz4) [reply spec/em_spec.rb]: What do you mean with let?

Comment (inhaq) [reply spec/em_spec.rb]: [Here you go](https://medium.com/@tomkadwill/all-about-rspec-let-a3b642e08d39)

File: spec/em_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#spec/em_spec.rb
+require './lib/enumerable-methods'  #=> add this
+
+
+
+RSpec.describe Enumerable do
```
Comment (inhaq) [inline spec/em_spec.rb:6]: Kindly put all tests within `RSpec.describe Enumerable do` block, you don't need to write it for each test

**Review by inhaq — CHANGES_REQUESTED** [review]
Good job, Team. Sorry, I just missed this minor tweak. Kindly look into it and update accordingly. 
Once updated, kindly resubmit for review.

**Review by inhaq — APPROVED** [review]
Awesome. It's 👍🏻
Good luck with the next project.

**Pull request merged**